exts/cesium.omniverse/mdl/cesium.mdl

```diff
@@ -0,0 +1,77 @@
+mdl 1.8;
+
+import ::anno::*;
+import ::state::*;
+import ::tex::*;
+
+using ::gltf::pbr import *;
+
+module [[
+    anno::version( 1, 0, 0),
+    anno::display_name("Cesium MDL functions")
+]];
+
+export float2 world_coordinate_2d(float2 min_world_xy=float2(-5000.0,-5000.0), float2 max_world_xy=float2(5000.0,5000.0))
```

Comment:
The default values don't seem to have any effect. They default to 0 and I have to go in and edit them separately.

Reply:
Oh weird, I can't reproduce this anymore. Maybe I had Fabric enabled by default and it didn't populate into USD correctly?